lib/iris/fileformats/_pyke_rules/fc_rules_cf.krb

```diff
@@ -1941,36 +1957,36 @@ fc_extras
         # Add it to the cube
         cube.add_aux_coord(coord, data_dims)
 
-        # Update the coordinate to CF-netCDF variable mapping.
+        # Make a list with names, stored on the engine, so we can find them all later.
         engine.provides['coordinates'].append((coord, cf_coord_var.cf_name))
 
 
     ################################################################################
-    def build_cell_measures(engine, cf_cm_attr, coord_name=None):
+    def build_cell_measures(engine, cf_cm_var):
         """Create a CellMeasure instance and add it to the cube."""
         cf_var = engine.cf_var
         cube = engine.cube
         attributes = {}
 
         # Get units
-        attr_units = get_attr_units(cf_cm_attr, attributes)
+        attr_units = get_attr_units(cf_cm_var, attributes)
 
-        data = _get_cf_var_data(cf_cm_attr, engine.filename)
+        data = _get_cf_var_data(cf_cm_var, engine.filename)
```

Comment:
For consistency, should this have a comment similar to this line? https://github.com/SciTools/iris/blob/1178de727b54e29efef7bc178feabfec1d8b164e/lib/iris/fileformats/_pyke_rules/fc_rules_cf.krb#L2018-L2019

Reply:
Ok, but "consistency" is a bit of a lost cause in this code IMHO.  Not least because we have far too much code duplication code, as here !
( N.B. We do still hope to replace all of this with Python code #3415  )